### ISSUE TITLE
Bank cost of upgrade

### ISSUE THREAD

**Issue by rabidworm** [Wynncraft/Issues#5649]
(just a heads up, tried to make sure there isn't a duplicate issue and im pretty sure there isn't but if there is I apologize).

tl;dr, buying bank upgrade costs less if you right click the first slot of the would be new row instead of any other slot when buying, AND only have the amount of EB needed to buy the previous row instead of the current row in your inventory.

Reproduction:
Hover over banks row upgrade, note its cost (5/9/etc). Remove all EB from inventory, right click on all slots of bank upgrade. When right clicking specifically the first slot int he row, the price is different and cheaper than the rest (identical to previous row's cost)

Let me know if you have any questions, I have reproduced this with all current rows I have bought in my bank (1 EB for first row instead of 5, 5 instead of 9 for next, 9 instead of 21 or however many for third row)
